### PR TITLE
Pull request to merge Italian translation (issue #6)

### PR DESCRIPTION
**Pull request by MorenoLaQuatra**
Hi, this is the PR to merge the Italian translation to the official repo (referencing issue: #6 ).

### CODE REVIEW

File: tutorials/IT/01_huggingface-hub-tour.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+Ecco alcune informazioni su Hugging Face Hub:
+
+- Ci sono oltre 30.000 modelli pubblici.
+- Ci sono modelli per il Natural Language Processing, Computer Vision, Audio/Parolat, e Reinforcement Learning!
+- Ci sono modelli per oltre 180 lingue.
+- Qualsiasi libreria di ML può sfruttare l'Hub: da TensorFlow e PyTorch a integrazioni avanzate con spaCy, SpeechBrain e altre 20 librerie.
+
+## Esplorare un modello
+
+Diamo il via all'esplorazione dei modelli. Puoi accedere a 30.000 modelli su [hf.co/models](http://hf.co/models). Vedrete [gpt2](https://huggingface.co/gpt2), uno dei modelli con più download. Clicchiamo su di esso.
```
Comment (FrancescoSaverioZuppichini) [inline tutorials/IT/01_huggingface-hub-tour.md:49]:
```suggestion
Diamo il via all'esplorazione dei modelli. Puoi accedere a 30.000 modelli su [hf.co/models](http://hf.co/models). Vedrete [gpt2](https://huggingface.co/gpt2), uno dei modelli con più downloads. Clicchiamoci sopra.
```

File: tutorials/IT/01_huggingface-hub-tour.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+
+Il sito vi porterà alla scheda del modello quando cliccate su di esso. Una scheda del modello è uno strumento che ne contiene la documentazion, fornendo informazioni utili sui modelli ed è essenziale per aumentarne la visibilità e la riproducibilità.
```
Comment (FrancescoSaverioZuppichini) [inline tutorials/IT/01_huggingface-hub-tour.md:51]:
```suggestion
Il sito vi porterà alla scheda del modello quando ci clicchiamo sopra. Una scheda del modello è uno pagina che ne contiene la documentazione fornendo informazioni utili sui modelli ed è essenziale per aumentarne la visibilità e la riproducibilità.
```

File: tutorials/IT/01_huggingface-hub-tour.md [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+
+L'interfaccia è composta da più parti, esaminiamole:
+
+[https://www.youtube.com/watch?v=XvSGPZFEjDY&feature=emb_imp_woyt](https://www.youtube.com/watch?v=XvSGPZFEjDY&feature=emb_imp_woyt)
+
+- In cima, puoi trovare diversi **tags** per cose come il compito svolto (*generazione di testo, classificazione di immagini*, ecc.), i frameworks utilizzati per la sua implementazione (*PyTorch*, *TensorFlow*, ecc.), la lingua del modello (*Inglese*, *Arabo*, *ecc.*), e la licenza (*es. MIT*).
```
Comment (FrancescoSaverioZuppichini) [inline tutorials/IT/01_huggingface-hub-tour.md:57]:
```suggestion
- In cima, puoi trovare diversi **tags** che descrivono la task del modello (*generazione di testo, classificazione di immagini*, ecc.), i frameworks utilizzati per la sua implementazione (*PyTorch*, *TensorFlow*, ecc.), la lingua del modello (*Inglese*, *Arabo*, *ecc.*), e la licenza (*es. MIT*).

File: tutorials/IT/01_huggingface-hub-tour.md [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+
+![](../../images/mode_card_tags.png)
+
+- Nella colonna di destra, puoi fare dei test con il modello direttamente nel browser usando l'*Inference API*. GPT2 è un modello di generazione di testo, quindi genererà un testo aggiuntivo dato un input iniziale. Provate a digitare qualcosa come "Era una giornata luminosa e soleggiata".
```
Comment (FrancescoSaverioZuppichini) [inline tutorials/IT/01_huggingface-hub-tour.md:61]:
```suggestion
- Nella colonna di destra, puoi testare il modello direttamente nel browser usando l'*Inference API*. GPT2 è un modello di generazione di testo, quindi genererà un testo aggiuntivo dato un input iniziale. Provate a digitare qualcosa come "Era una giornata luminosa e soleggiata".
```

File: tutorials/IT/01_huggingface-hub-tour.md [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+
+            ```python
+            git lfs track "*.your_extension"
+            ```
+
+            ```python
+             git lfs install
+            ```
+
+    4. Aggiungi i tuoi file alla repository. I file dipendono dal framework/librerie che stai usando. Nel complesso, ciò che è importante è che tu fornisca tutti gli artefatti richiesti per caricare il modello. Per esempio:
```
Comment (FrancescoSaverioZuppichini) [inline tutorials/IT/01_huggingface-hub-tour.md:129]:
```suggestion
    4. Aggiungi i tuoi file alla repository. I file dipendono dal frameworks/librerie che stai usando. Nel complesso, ciò che è importante è che tu fornisca tutti gli artefatti richiesti per caricare il modello. Per esempio:
   ```

File: tutorials/IT/01_huggingface-hub-tour.md [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
+        1. Per TensorFlow, potresti voler caricare un file SavedModel or `h5` file.
+        2. Per PyTorch, di solito, è un `pytorch_model.bin`.
+        3. Per Scikit-Learn, di solito è un file `joblib` file.
```
Comment (FrancescoSaverioZuppichini) [inline tutorials/IT/01_huggingface-hub-tour.md:132]:
```suggestion
        3. Per Scikit-Learn, di solito è un file `joblib`.
        ```

File: tutorials/IT/02_ml-demos-with-gradio.ipynb [hunk 7]
```diff
@@ -0,0 +1,497 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "D_Iv1CJZPekG"
+   },
+   "source": [
+    "**Obiettivi di apprendimento:** L'obiettivo di questo tutorial è imparare come\n",
+    "\n",
+    "1. Costruire una rapida demo per il tuo modello di machine learning in Python usando la libreria `gradio`.\n",
+    "2. Ospitre le demo gratuitamente con Hugging Face Spaces\n",
```
Comment (FrancescoSaverioZuppichini) [inline tutorials/IT/02_ml-demos-with-gradio.ipynb:40]:
```suggestion
    "2. Pubblicare le demo gratuitamente con Hugging Face Spaces\n",
```

File: tutorials/IT/02_ml-demos-with-gradio.ipynb [hunk 8]
```diff
@@ -0,0 +1,497 @@
...
+    "gr.Interface(fn=greet, inputs=\"text\", outputs=\"text\").launch();"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "0TyTGpSsb7bs"
+   },
+   "source": [
+    "L'esecuzione del codice mostrato sopra dovrebbe produrre una semplice interfaccia grafica all'interno di questo ambiente di sviluppo. Quesat ti permetterà di digitare input di esempio e vedere l'output restituito dalla vostra funzione. \n",
```
Comment (FrancescoSaverioZuppichini) [inline tutorials/IT/02_ml-demos-with-gradio.ipynb:181]:
```suggestion
    "L'esecuzione del codice mostrato sopra dovrebbe produrre una semplice interfaccia grafica all'interno di questo ambiente di sviluppo. Questa ti permetterà di digitare input di esempio e vedere l'output restituito dalla vostra funzione. \n",
```

File: tutorials/IT/README.md [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+</aside>
+
+## **Il catalogo dei corsi**
+
+### 1️⃣ Un tour attraverso l'Hub di Hugging Face
+
+> In questo tutorial potrai:
+>
+> - Esplorare gli oltre 30.000 modelli condivisi su Hub.
+> - Apprendere modi efficienti per trovare modelli e set di dati adatti al tuo task.
```
Comment (FrancescoSaverioZuppichini) [inline tutorials/IT/README.md:23]:
```suggestion
> - Apprendere modi efficaci per trovare modelli e set di dati adatti al tuo task.
```
Penso sia meglio  :)

File: tutorials/IT/03_getting-started-with-transformers.ipynb [hunk 10]
```diff
@@ -0,0 +1,9727 @@
...
+        "sentiment_pipeline = pipeline('text-classification')"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "4-JvPV-IDvqc"
+      },
+      "source": [
+        "Quando si esegue questo codice, si vedrà un messaggio su quale modello Hub viene usato di default. In questo caso, la funzione `pipeline()` carica il modello `distilbert-base-uncased-finetuned-sst-2-english`, che è una piccola variante BERT addestrata su [SST-2](https://paperswithcode.com/sota/sentiment-analysis-on-sst-2-binary) che è un dataset di analisi del sentimento nei testi (sentiment analysis)."
```
Comment (FrancescoSaverioZuppichini) [inline tutorials/IT/03_getting-started-with-transformers.ipynb:478]:
```suggestion
"Quando si esegue questo codice, si vedrà un messaggio che ci dice quale modello caricato dall'Hub viene usato di default. In questo caso, la funzione `pipeline()` carica il modello `distilbert-base-uncased-finetuned-sst-2-english`, che è una piccola variante BERT addestrata su [SST-2](https://paperswithcode.com/sota/sentiment-analysis-on-sst-2-binary) che è un dataset di analisi del sentimento nei testi (sentiment analysis)."
```

File: tutorials/IT/01_huggingface-hub-tour.md [hunk 11]
```diff
@@ -0,0 +1,227 @@
...
+Obiettivi di apprendimento:
+
+- Conoscere ed esplorare gli oltre 30.000 modelli condivisi su Hub.
+- Apprendere modi efficienti per trovare modelli e set di dati adatti al tuo task.
+- Apprendere come contribuire e lavorare in modo collaborativo.
+- Esplorare le demo delle tecniche di Apprendimento Automatico (Machine Learning) create dalla comunità.
+
+**Formato:** Laboratorio breve o da casa
+
+**Pubblico:** Studenti con qualsiasi conoscenza pregressa interessati ad utilizzare modelli esistenti o a condividere i propri.
```
[the remark this answers is not in the document]
Comment (MorenoLaQuatra) [reply tutorials/IT/01_huggingface-hub-tour.md:28]: I agree, thank you so much for the revision ;)
Concordo, grazie mille per la revisione! ;)

**Comment by MorenoLaQuatra** [comment]
I committed the very useful corrections (and the suggestion) by @FrancescoSaverioZuppichini .

**Comment by MorenoLaQuatra** [comment]
I committed the very useful corrections (and the suggestion) by @FrancescoSaverioZuppichini .

**Review by FrancescoSaverioZuppichini — APPROVED** [review]
Thanks! LGTM!

**Comment by omarespejel** [comment]
Grazie @MorenoLaQuatra e @FrancescoSaverioZuppichini!

**Pull request merged**